File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/SKSize.java
package br.com.insanitech.spritekit;

import br.com.insanitech.spritekit.opengl.model.GLSize;

/**
 * Created by anderson on 7/4/15.
 */
public class SKSize extends GLSize {
    public SKSize() {
        super();
    }

    public SKSize(float width, float height) {
        super(width, height);
    }

    SKSize(GLSize other) {
        width = (other.width);
        height = (other.height);
    }

    @Override
    public String toString() {
        return "{ w: " + width + ", h: " + height + " }";
    }
}

File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/SKSize.kt
package br.com.insanitech.spritekit

import br.com.insanitech.spritekit.opengl.model.GLSize

/**
 * Created by anderson on 7/4/15.
 */
class SKSize : GLSize {
    constructor() : super() {}

    constructor(width: Float, height: Float) : super(width, height) {}

    internal constructor(other: GLSize) {
        width = other.width
        height = other.height
    }

    override fun toString(): String {
        return "{ w: $width, h: $height }"
    }
}
